### ISSUE TITLE
Middle mouse click handling

### ISSUE THREAD

**Issue by adamschwartz** [adamschwartz/chrome-tabs#9]
See https://github.com/adamschwartz/chrome-tabs/pull/6

**Comment by 190n** [comment]
+1

**Comment by amovah** [comment]
I had done this issue long time ago, but I can do it again.
I will fix this issue

**Comment by adamschwartz** [comment]
Yeah @amovah, sorry about that. I had [planned to re-make your changes](https://github.com/adamschwartz/chrome-tabs/pull/6#issuecomment-97635870) but haven’t gotten to it yet. Thx!
